main.go

```diff
@@ -95,6 +96,7 @@ func main() {
 	flags.StringVar(&opt.LocalData, "local-data", opt.LocalData, "Path to testgrid data from local disk")
 	flags.StringVar(&opt.DSN, "database-dsn", os.Getenv("SIPPY_DATABASE_DSN"), "Database DSN for storage of some types of data")
 	flags.StringArrayVar(&opt.OpenshiftReleases, "release", opt.OpenshiftReleases, "Which releases to analyze (one per arg instance)")
+	flags.StringArrayVar(&opt.OpenshiftArchitectures, "architecture", opt.OpenshiftArchitectures, "Which architectures to analyze (one per arg instance)")
```

Comment:
Nit: I'd suggest --arch for the arg just for brevity.

Should this be plugged into the fetchdata script?